fuel-client/assets/schema.sdl

```diff
@@ -98,21 +114,23 @@ type CoinConnection {
 	"""
 	A list of edges.
 	"""
-	edges: [CoinEdge]
+	edges: [CoinEdge!]!
 }
+
 """
 An edge in a connection.
 """
 type CoinEdge {
-	"""
-	The item at the end of the edge
-	"""
-	node: Coin!
 	"""
 	A cursor for use in pagination
 	"""
 	cursor: String!
+	"""
+	"The item at the end of the edge
```

Comment:
This kind of looks like a bug in async-grapqhl, why do all these connection edges start with quotes

```suggestion
	"The item at the end of the edge
```